src/org/rust/lang/structure/RustFileTreeElement.kt

```diff
@@ -18,6 +16,14 @@ class RustStructureViewTreeElement(file: RustFileImpl) : PsiTreeElementBase<Rust
                 o.acceptChildren(this)
             }
 
+            override fun visitEnumItem(o: RustEnumItem) {
+                result += (RustStructureViewElement(o))
```

Comment:
Good to know. I've never written Kotlin before ;)

I'm actually not yet convinced that the recursive traversal here makes that much sense, but I'll keep it in mind. Thanks!